### DISCUSSION TITLE
Not able to figure it out where i can get SAD_SVG_URI...

### DISCUSSION THREAD

**Discussion by ArlisVilipold** [Cyfrin/foundry-full-course-cu#2224] [Q&A]
![SVG_URI](https://github.com/user-attachments/assets/6d89b0b0-b3fd-4370-b93a-de87cc5b355c)

In 16.Debug your smart contract video 02:48 Patrick says "i'm going to cheat a lil bit" and then copies the json from browser address.. 
I'm not able to figure it out where i could find this.

**Comment by EngrPips** [comment]
Hello @ArlisVilipold, Here is the link https://github.com/Cyfrin/foundry-nft-cu/blob/main/test/MoodNftTest.sol to the test file of Patrick. You should be able to see every information of the test in the link I shared.
